### PR TITLE
remove Help Server unofficial property

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #22127

### Summary of Changes
redo of #22133  - removed Help Server help.joomla.fr cause is unofficial property

### Testing Instructions
check help site

### Expected result
no more french Help Server

### CODE REVIEW

File: libraries/src/Form/Field/HelpsiteField.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
  *
  * @copyright  Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.
  * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ * @deprecated  4.0 To be removed
```
Comment (mbabker) [inline libraries/src/Form/Field/HelpsiteField.php]: `@deprecated` annotations should be on the class element, not the file element.

Comment (alikon) [reply libraries/src/Form/Field/HelpsiteField.php]: sorry but i'm really a dumb on this matter

**Comment by Quy** [comment]
Deprecate `JGLOBAL_HELPREFRESH_BUTTON`

**Comment by alikon** [comment]
done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 06181e580785857aab1ced824de6424ff4c1a5e3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24927">issues.joomla.org/tracker/joomla-cms/24927</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 06181e580785857aab1ced824de6424ff4c1a5e3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24927">issues.joomla.org/tracker/joomla-cms/24927</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24927">issues.joomla.org/tracker/joomla-cms/24927</a>.</sub>

**Comment by HLeithner** [comment]
@alikon  Didn't you need to rebuild the css file?

**Comment by alikon** [comment]
iirc i've runned `genetatecss.php`  before to sumbit the pr

**Comment by HLeithner** [comment]
there is no new css file for protostar

**Comment by alikon** [comment]
bad memory or bad settings ?
surely both
:smile: 
fixed now 
double check please

**Comment by HLeithner** [comment]
thx

**Comment by alikon** [comment]
thanks
@HLeithner 
don't forget to look at https://github.com/joomla/update.joomla.org/pull/111 too :smiley:

**Pull request merged**